packages/babel-preset-gatsby-package/index.js

```diff
@@ -4,23 +4,20 @@ function preset(context, options = {}) {
   const { browser = false, debug = false, nodeVersion = `8.0` } = options
   const { NODE_ENV, BABEL_ENV } = process.env
 
-  const IS_PRODUCTION = (BABEL_ENV || NODE_ENV) === `production`
   const IS_TEST  = (BABEL_ENV || NODE_ENV) === `test`
 
   const browserConfig = {
     useBuiltIns: false,
     targets: {
-      browsers: IS_PRODUCTION
-        ? [`last 4 versions`, `safari >= 7`, `ie >= 9`]
-        : [`last 2 versions`, `not ie <= 11`, `not android 4.4.3`],
+      browsers: [`last 2 versions`, `not ie <= 11`, `not android 4.4.3`],
```

Comment:
Will this change the client-side JS that's output during a build?

Reply:
No that's babel-preset-gatsby :)